code/game/machinery/washing_machine.dm

```diff
@@ -44,7 +45,21 @@
 	for(var/obj/item/clothing/suit/hooded/ian_costume/IC in contents)
 		new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/corgi(src)
 		qdel(IC)
-
+	
+	for(var/obj/item/weapon/paper/P in contents)
+		if(crayon)
+			var/dye_color
+			if(istype(crayon,/obj/item/toy/crayon))
+				var/obj/item/toy/crayon/CR = crayon
+				dye_color = CR.colourName

```

Comment:
It's better to use actual colors rather than color names, even if byond does parse the names properly.